src/xia2/Applications/xia2setup.py

```diff
@@ -202,7 +202,11 @@ def visit(directory, files):
                     "Ignoring %s (Registry can not find format class)" % full_path
                 )
                 continue
-            elif format_class.ignore():
+            elif getattr(
+                format_class, "is_abstract", getattr(format_class, "ignore")
+            )():
+                # temporary workaround while not all format classes have the
+                # is_abstract method - https://github.com/xia2/xia2/issues/567
```

Comment:
```suggestion
            elif format_class.is_abstract():
```
So, given discussion, I think it's safe to not check ignore now? the only case I can think of that would not work here is if someone had a 
- custom format class
- that they used only with xia2. 

I think that set is probably zero?